lib/components/tooltip.js

```diff
@@ -127,6 +134,38 @@ const addNpmLink = (data) => {
   return npm;
 };
 
+const addAtomLink = (data) => {
+  if (!data || !data.name) {
+    return undefined;
+  }
+
+  const dirlink = new Directory(atom.workspace.getActiveTextEditor().buffer.file.getParent().path+"/node_modules/"+data.name)
+  const existDirlink = dirlink.existsSync()
+
+  const icon = el('i');
+  setAttr(icon, {
+    className: 'icon icon-file-directory atom-icon',
+  });
+
+  const iconWrapper = el('span', icon);
+  setAttr(iconWrapper, {
+    className: 'tab selected',
+  });
+
+  const atom_el = el('a', iconWrapper);
+  setAttr(atom_el, {
+    className: 'btn' + (existDirlink ? '' : ' btn-disabled'),
+    title: data.name,
```

Comment:
What do you think about hiding the button entirely when the package is not installed? That or have a different title so the users know that for using this button they need to actually have the package installed.

Reply:
Indeed, it was my plan B.